### PR TITLE
C2C-83 Set default locale to 'fr'

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

File: configuration/concepts/commons.csv [hunk 1]
```diff
@@ -126,11 +128,13 @@ a82e292b-7bd3-4bf1-b6d8-d89c8b8cabff,,,Two,Deux,Two,Deux,,,Misc,N/A,,,,,,,,,,,
...
 d29d0005-5e23-48cd-86ac-1ff9cd36dd13,,,Four,Quatre,Four,Quatre,,,Misc,N/A,,,,,,,,,,,
 3098581b-4ce3-4064-8e87-facb1d7a5ff5,,,Five plus,Cinq plus,Five plus,Cinq plus,,,Misc,N/A,,,,,,,,,,,
-5f2ee7bc-8dcf-4617-b31c-2ac6c07617b5,,,Zero,Zéro,0,0,,,Misc,N/A,,,,,,,,,,,
+5f2ee7bc-8dcf-4617-b31c-2ac6c07617b5,,,Zero,Zero,0,0,,,Misc,N/A,,,,,,,,,,,
 e7c0fefc-96e8-4ccb-b24b-92ec2024f579,,CIEL:1362,One plus,Un plus,+,+,,,Misc,N/A,,,,,,,,,,,
 9f14f988-be10-419b-b947-0508664b7ef2,,CIEL:1363,Two plus,Deux plus,++,++,,,Misc,N/A,,,,,,,,,,,
 a979848e-4b09-4319-bdd5-ce7d38e0b686,,CIEL:1364,Three plus,Trois plus,+++,+++,,,Misc,N/A,,,,,,,,,,,
 30c7d366-2192-45bf-a0df-112258bc0c1e,,CIEL:1365,Four plus,Quatre plus,++++,++++,,,Misc,N/A,,,,,,,,,,,
-c4b48a7a-7c13-4e7d-89ad-1df68c5c6802,,,Number of Visit,Nombre de visites,Number of Visit,Nombre de visites,,,Misc,Coded,,One;Two;Three;Four;Five plus,,,,,,,,,
+c4b48a7a-7c13-4e7d-89ad-1df68c5c6802,,,Number of Visit,Nombre de visites,Number of Visit,Nombre de visites,,,Misc,Coded,,Une;Deux;Trois;Quatre;Cinq plus,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: I think this should be renamed:
'Visit number" / "Numéro de la visite"

File: configuration/concepts/patient_document.csv [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Description:en,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1000
-81f330b5-3f10-11e4-adec-0800271c1b75,,,Document,Document,,Misc,Text,,,,,,,,,,,
-c4694ad6-3f10-11e4-adec-0800271c1b75,,,Patient file,Patient file,,Misc,N/A,,,,,,,,,,,
-c46896fd-3f10-11e4-adec-0800271c1b75,,,Patient Document,Patient Document,,ConvSet,N/A,,,Patient file,,,,,,,,
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1000
+81f330b5-3f10-11e4-adec-0800271c1b75,,,Document,Document,Document,Document,,Misc,Text,,,,,,,,,,,
+c4694ad6-3f10-11e4-adec-0800271c1b75,,,Patient file,Dossier patient,Patient file,Dossier patient,,Misc,N/A,,,,,,,,,,,
+c46896fd-3f10-11e4-adec-0800271c1b75,,,Patient Document,Document patient,Patient Document,Document patient,,ConvSet,N/A,,,Dossier patient,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/patient_document.csv]: Did you try the document upload? (just to make sure)

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: configuration/concepts/commons.csv (hunk 1 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/patient_document.csv (hunk 2 above)
Comment (reagan-meant) [reply configuration/concepts/patient_document.csv]: Fixed...The french name expected is still `Patient Document`

**Review by rbuisson — APPROVED** [review]

**Pull request merged**